### ISSUE TITLE
Might be a typo?

### ISSUE THREAD

**Issue by petroskey** [Lezcano/expRNN#4]
Hi @Lezcano,

Thank you for sharing your great codes and reply the other day.

When I try to run 2_mnist.py, I got  an error. 
I am not sure but considering your another project 1_copying.py, which worked as is, the following part of the 2_mnist.py might be typo I suppose.

After line 114, should it be as follows?

    if optim_orth:
        optim_orth.zero_grad()
    optim.zero_grad()

    loss.backward()

    if optim_orth:
        orthogonal_step(model, optim_orth)
    optim.step()

Thank you in advance.

**Comment by Lezcano** [comment]
Absolutely. I don't know where was that introduced, but it's clearly as you say.

I have some updated code to perform all these things in an (arguably) cleaner way. I'll update everything when I release the follow-up paper that was accepted to NeurIPS. Hopefully I'll tod that sometime this week.

**Comment by petroskey** [comment]
Thank you for your reply, Lezcano.

Congratulations for your next paper!
I am looking forward to seeing it.
I will find it in NIPS page.

Thank you.

2019年9月17日(火) 17:34 Lezcano <notifications@github.com>:
